scanners/garak/garak_none.py

```diff
@@ -18,12 +19,12 @@
 
 
 @dataclass
+# pylint: disable=too-many-instance-attributes
 class GarakConfig:
-    model_type: str
-    model_name: str = field(default="test_model")
-    probe_spec: str = field(default="all")  # all or a list of probes like "probe1,probe2"
-    garak_executable_path: str = field(default="/usr/local/bin/garak")
-    generators: Dict[str, Any] = field(default_factory=dict)
+    parameters: Optional[Dict[str, Any]] = field(default_factory=dict)
```

Comment:
The `parameters` property is not `Optional`, right? It should always be initialized

Reply:
Sorry I just noticed your comment after the merge. You're correct, so I'll create a new PR for it.